pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1008,6 +1008,11 @@
     )
     private boolean disableBrokerInterceptors = true;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "List of interceptors for payload processing.")
+    private String brokerEntryPayloadProcessor;
```

Comment:
It could be a list of interceptors, so the type should be Set<String>.

Reply:
I will fix it. Thanks for catching this.